### ISSUE TITLE
Splash screen idles unnecessarily

### ISSUE THREAD

**Issue by Vavassor** [Vavassor/Tusky#81]
If you remove the splash screen time delay, it actually just shows a black screen. Because it takes longer to load the splash screen animation itself, than to actually load the MainActivity.

So, the splash screen is sort of a lie right now. It loads itself, does a bit of nothing, then loads the main activity. Instead of the purpose of a loading screen. which is to show something while the MainActivity loads asynchronously.

**Comment by Gargron** [comment]
I like seeing th splash screen. Maybe you could start the network requests in it though instead of only starting them when MainActivitiy loads

**Comment by pierreduchemin** [comment]
That splash screen seems to be a bit overkill to me, but the idea of displaying a nice animation while initializing content is really good.
Maybe we could switch to something like that : https://saulmm.github.io/avoding-android-cold-starts

**Comment by daycode** [comment]
I agree with @pierreduchemin plus, the first loading of the content at start take sometimes few seconds, so it would be interesting to load it in background when the spash screen is display.

**Comment by charlag** [comment]
My point on this is that splash screen should be as short as possible - and we should follow best practices here https://www.bignerdranch.com/blog/splash-screens-the-right-way/
As Splash Activity and Timeline Activity are different activities we cannot 'preheat' data without saving timeline to a DB.
Here's the thing: if we're going to use DB, we can show saved toots in timeline instead of splash screen and I think it's better.

**Comment by Vavassor** [comment]
I replaced the splash screen back in commit 3c20f7a0c46cc6e481f3be8d4bf0650444e9d404 with a static one. Any method I could find of adding animation that required making views created a similar load delay as the prior animated screen. Ended up having to wait for that to get created instead of doing the work to load the main page. Whereas the static version just loads one ```Drawable```, which I suppose goes so fast because it sidesteps the whole layout system.

There's a possibility of using ```AnimatedVectorDrawableCompat``` to make an animation of some kind, but I haven't taken the time to check that out.

It is actually loading all the visual elements of the main activity right now during the splash screen, but it doesn't wait for all the network requests to return. So the timeline still fills in after the splash screen.

Anyhow, my initial complaint is done so this is closed for now.
